.concourse/pipeline.yaml.gomplate

```diff
@@ -343,8 +343,7 @@ jobs:
       image_resource:
         type: registry-image
         source:
-          repository: thulioassis/bazel-docker-image
-          tag: 2.0.0
+          repository: splatform/catapult
```

Comment:
Shouldn't this be pinned to _something_? Otherwise when we ship a new version (e.g. for kubecf 2.8) it will break older pipelines (e.g. kubecf 2.4).

Reply:
I agree. Found the pipeline that releases catapult images, started tagging there:
  https://github.com/SUSE/cloudfoundry/pull/180
  https://concourse.suse.de/teams/main/pipelines/catapult

Added a tag and anchored it throughout the pipeline.